packages/main/src/services/local-library/interfaces.ts

```diff
@@ -0,0 +1,22 @@
+import { NuclearStream } from '@nuclear/core';
+import LocalFolder from './model/LocalFolder';
+
+export interface NuclearLocalMeta {
```

Comment:
Can you try to make it compatible with the types I'm working on for the search plugins?

i. e. https://github.com/nukeop/nuclear/blob/feature/search-providers/packages/core/src/plugins/plugins.types.ts

Reply:
I see you just merge your branch :) :) :). I will refactor this stuff this week.

Reply:
Nice, i tried to move a lot of functionality to typescript. Some tests are broken because of this, not sure why. We're at 17% TS now

Reply:
I am still a little bit confuse with the model.
for example in the queue the thumbnail has to be named thumb, but in the library view it has to be thumbnail.
artist is sometimes a string sometimes an object.

Reply:
This is a consequence of the different apis returning different things. I was trying to standardize what they return while retaining as much information as possible, and preserving naming etc. If you have a better idea how to organize these structures, by all means improve them.

Reply:
Ok, i will keep it simple for now.

What do you think of improving a bit the Track struct. We could use getter to access thumbnail or artist for example. (in another PR)